ibridges/cli/__main__.py

```diff
@@ -0,0 +1,33 @@
+"""Entry point for CLI interface."""
+
+import argparse
+import sys
+
+from ibridges.cli.other import CLI_BULTIN_COMMANDS
```

Comment:
For the docs: Can you write a small tutorial or demo/template plugin "hello world" to show how to extend the functionality. It does not necessarily need to be part of this PR, but certainly something for a next round.

Reply:
I will make it part of this PR.

Reply:
Done